File: 01-Arrays/java/src/Main.java
import java.util.ArrayList;
import java.util.Vector;

public class Main {
    public static void main(String[] args) {

//        Built-in Array
        ArrayList<Integer> list = new ArrayList<>();
        Vector<Integer> list_v = new Vector<>();

//        Custom Array
        System.out.println("Custom Array");

        Array numbers = new Array(3);
        numbers.insert(-1);
        numbers.insert(0);
        numbers.insert(1);
        numbers.insert(10);
        numbers.insert(20);
        numbers.insert(30);
        numbers.insert(40);
        numbers.print();
        numbers.removeAt(6);
        numbers.print();

        System.out.println("--------");

    }

}

File: 01-Arrays/cpp/main.cpp
#include <iostream>
#include "Array.h"

int main() {

//    Custom Array
    cout << "Custom Array" << endl;

    Array numbers(3);

    numbers.insert(100);
    numbers.insert(400);
    numbers.insert(520);
    numbers.print();

    Array other(5);
    other.insert(3);
    other.insert(400);
    other.insert(100);
    other.print();

    Array inter = numbers.intersect(other);
    inter.print();

    cout << "--------" << endl;


    return 0;
}
